src/index.tsx

```diff
@@ -177,34 +185,44 @@ export default class OtpInputs extends PureComponent<Props, State> {
       selectTextOnFocus,
       testIDPrefix,
       unfocusedBorderColor,
+      forceLTR,
+      isRTL,
+      debug,
+      placeholder,
     } = this.props
     const { otpCode } = this.state
     const iterationArray = Array<number>(numberOfInputs).fill(0)
 
-    return iterationArray.map((_, index) => (
-      <OtpInput
-        autoCapitalize={autoCapitalize}
-        clearTextOnFocus={clearTextOnFocus}
-        containerStyles={inputContainerStyles}
-        focusStyles={focusStyles}
-        error={!!errorMessage}
-        focusedBorderColor={focusedBorderColor}
-        handleBackspace={(event: TextInputOnKeyPressEventData) =>
-          this._handleBackspace(event, index)
-        }
-        inputStyles={inputStyles}
-        key={index}
-        secureTextEntry={secureTextEntry}
-        keyboardType={keyboardType}
-        ref={this.inputs[index]}
-        selectTextOnFocus={selectTextOnFocus}
-        textErrorColor={inputTextErrorColor}
-        unfocusedBorderColor={unfocusedBorderColor}
-        updateText={(event: TextInputOnChangeEventData) => this._updateText(event, index)}
-        value={otpCode[index]}
-        testID={`${testIDPrefix}-${index}`}
-      />
-    ))
+    return iterationArray.map((_, index) => {
+      if(forceLTR && isRTL){
+        index = (numberOfInputs - 1)  - index
```

Comment:
Please create a new variable instead of overriding `index` which comes from `map` function.
```
let inputIndex = index
if(...){
...
}
return (...)
```